libs/aws/langchain_aws/llms/sagemaker_endpoint.py

```diff
@@ -263,6 +268,33 @@ def transform_output(self, output: bytes) -> str:
     .. _boto3: <https://boto3.amazonaws.com/v1/documentation/api/latest/index.html>
     """
 
+    deployment_config: Optional[Dict] = None
```

Comment:
Rather than keeping this as an open-ended dictionary, how about encoding this as a pydantic model, with allowed additions.